src/agentscope/service/file/common.py

```diff
@@ -201,3 +202,61 @@ def move_directory(
             status=ServiceExecStatus.ERROR,
             content=error_message,
         )
+
+
+def show_directory(directory_path: str) -> ServiceResponse:
```

Comment:
"show_directory" is more like "pwd" rather than "ls -a", how about `list_directory_contents`